### PR TITLE
AB#194768 new menu structure

### PR DESCRIPTION
**Pull request by johniak**
[AB#194768](https://unicef.visualstudio.com/4e044e8d-bc28-4768-8074-f80ff6e4a0e5/_workitems/edit/194768)

### CODE REVIEW

**Review by pavlo-mk — APPROVED** [review]

File: frontend/src/components/core/Drawer/Drawer.tsx [hunk 1]
```diff
@@ -141,6 +160,16 @@ export const Drawer = ({
     }
   }, [backendVersion, frontendVersion, showMismatchedDialog]);
 
+  let notActiveBar = null;
+  if (
+    !isAllPrograms &&
+    programResults?.data?.program?.status !== ProgramStatus.Active &&
+    programResults?.data?.program?.status !== undefined &&
```
Comment (mmaciekk) [inline frontend/src/components/core/Drawer/Drawer.tsx:167]: `const programStatus = programResults?.data?.program?.status;
const isActive = programStatus === ProgramStatus.Active;
const isDefined = programStatus !== undefined && programStatus !== null;

let notActiveBar = null;
if (!isAllPrograms && !isActive && isDefined) {
  notActiveBar = <ProgramNotActiveBar data-cy="program-inactive-subheader">program inactive</ProgramNotActiveBar>;
`}``

File: frontend/src/components/core/Drawer/Drawer.tsx [hunk 2]
```diff
@@ -141,6 +160,16 @@ export const Drawer = ({
...
+    programResults?.data?.program?.status !== null
```
Comment (mmaciekk) [inline frontend/src/components/core/Drawer/Drawer.tsx:168]: `const programStatus = programResults?.data?.program?.status;
const isActive = programStatus === ProgramStatus.Active;
const isDefined = programStatus !== undefined && programStatus !== null;

let notActiveBar = null;
if (!isAllPrograms && !isActive && isDefined) {
  notActiveBar = <ProgramNotActiveBar data-cy="program-inactive-subheader">program inactive</ProgramNotActiveBar>;
}`

File: frontend/src/components/core/Drawer/DrawerItems.tsx [hunk 3]
```diff
@@ -90,39 +102,34 @@ export const DrawerItems = ({
   if (permissions === null || !businessAreaData) return null;
 
   const prepareMenuItems = (items: MenuItem[]): MenuItem[] => {
-    const updatedMenuItems = [...items];
+    let updatedMenuItems = [...items];
     const getIndexByName = (name: string): number =>
       updatedMenuItems.findIndex((item) => item?.name === name);
-    const cashAssistIndex = getIndexByName('Cash Assist');
-    const programDetailsIndex = getIndexByName('Programme Details');
+    const programDetailsIndex = getIndexByName('Program Details');
```
Comment (mmaciekk) [inline frontend/src/components/core/Drawer/DrawerItems.tsx:108]: we don't say Programme now?

File: frontend/src/components/core/Drawer/menuItems.tsx [hunk 4]
```diff
@@ -34,62 +34,95 @@ export type MenuItem = {
...
+    name: 'Registration Data Import',
+    href: '/registration-data-import-for-people',
+    selectedRegexp: /^\/registration-data-import.*$/,
+    icon: <AutorenewIcon />,
+    isSocialWorker: true,
+    permissions: [PERMISSIONS.RDI_VIEW_DETAILS, PERMISSIONS.RDI_VIEW_LIST],
+    scopes: [SCOPE_PROGRAM],
+  },
+  {
+    name: 'Program Population',
```
Comment (mmaciekk) [inline frontend/src/components/core/Drawer/menuItems.tsx:72]: same here. Do we spell it Program now?

File: frontend/src/components/programs/ProgramDetails/ProgramDetails.tsx [hunk 5]
```diff
@@ -138,7 +138,7 @@ export const ProgramDetails = ({
           </Grid>
         </Grid>
         <NumberOfHouseHolds>
-          <LabelizedField label={t('Total Number of Households')}>
+          <LabelizedField label={t('Program size')}>
```
Comment (mmaciekk) [inline frontend/src/components/programs/ProgramDetails/ProgramDetails.tsx:141]: I guess we do spell it Program now...

File: frontend/src/containers/tables/people/PeopleListTable/PeopleListTable.tsx [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+  useAllIndividualsForPopulationTableQuery,
+} from '@generated/graphql';
+import { TableWrapper } from '@components/core/TableWrapper';
+import { dateToIsoString } from '@utils/utils';
+import { UniversalTable } from '../../UniversalTable';
+import { useBaseUrl } from '@hooks/useBaseUrl';
+import { headCells } from './PeopleListTableHeadCells';
+import { PeopleListTableRow } from './PeopleListTableRow';
+
+interface IndividualsListTableProps {
```
Comment (mmaciekk) [inline frontend/src/containers/tables/people/PeopleListTable/PeopleListTable.tsx:17]: PeopleListTableProps

File: frontend/src/containers/tables/people/PeopleListTable/PeopleListTable.tsx [hunk 7]
```diff
@@ -0,0 +1,70 @@
...
+  canViewDetails: boolean;
+  choicesData: HouseholdChoiceDataQuery;
+}
+
+export function PeopleListTable({
+  businessArea,
+  filter,
+  canViewDetails,
+  choicesData,
+}: IndividualsListTableProps): React.ReactElement {
```
Comment (mmaciekk) [inline frontend/src/containers/tables/people/PeopleListTable/PeopleListTable.tsx:29]: PeopleListTableProps

**Review by mmaciekk — APPROVED** [review]

File: frontend/src/components/core/Drawer/menuItems.tsx (hunk 4 above)
Comment (johniak) [reply frontend/src/components/core/Drawer/menuItems.tsx:72]: ticket says no

File: frontend/src/components/programs/ProgramDetails/ProgramDetails.tsx (hunk 5 above)
Comment (johniak) [reply frontend/src/components/programs/ProgramDetails/ProgramDetails.tsx:141]: ticket says yes

File: frontend/src/containers/tables/people/PeopleListTable/PeopleListTable.tsx (hunk 6 above)
Comment (johniak) [reply frontend/src/containers/tables/people/PeopleListTable/PeopleListTable.tsx:17]: nice, I will fix that

**Pull request merged**